hypothesis-python/src/hypothesis/extra/_array_helpers.py

```diff
@@ -335,7 +335,7 @@ def _hypothesis_parse_gufunc_signature(signature):
                 names_out = {n.strip("?") for n in result_shape}
                 if name.strip("?") in (names_out - names_in):
                     raise InvalidArgument(
-                        "The {name!r} dimension only appears in the output shape, and is "
+                        f"The {name!r} dimension only appears in the output shape, and is "
                         "not frozen, so the size is not determined ({signature=})."
```

Comment:
It looks like we'll need a `f` prefix in the second line as well, for `{signature=}`.